src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_DraftApprenticeship.cshtml

```diff
@@ -460,4 +459,27 @@
 
         return indexed_array;
     }
+
+    
+    function onFormSubmitted(e) {
+        e.preventDefault();
+        if ($("#draftApprenticeshipSection1").hasClass("das-hide") === false)
+            continueJourney();
+        else return;
```

Comment:
Does returning in this function end up calling the default form submit behaviour?

Reply:
yup